pkg/libcose/Makefile.libcose

```diff
@@ -0,0 +1,3 @@
+MODULE := libcose
```

Comment:
Why here `:=` and below `SUBMODULES =`? I think I saw more often the opposite. Though, there's a specific use for each one so I prefer to have a strong reason for using them.

Reply:
Both `:=` and `=` are used equally with `MODULE` in packages. Both work in this case. Anything you want me to do with them?

Reply:
As there is nothing to expand in these variables, in this case they are equivalent. If you want to have it consistent, I can change it so that both are set with `:=`

Reply:
Ok, just leave it as it is. I agree here it doesn't matter, just for the sake of consistency at least there's a record.